compose.deploy.yml

```
@@ -0,0 +1,84 @@
+  # ---------------------------------------------------------------------------
+  # ToM Server — containerised
+  # Config is loaded from /etc/twake/chat/tom/config.yaml (see docker.md)
+  # ---------------------------------------------------------------------------
+  tom:
+    image: linagora/tom-server
+    build:
+      context: .
+      dockerfile: apps/tom-server/Dockerfile
+    restart: unless-stopped
+    depends_on:
+      synapse: { condition: service_healthy }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)